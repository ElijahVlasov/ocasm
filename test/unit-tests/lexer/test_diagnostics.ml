open! Import

let () = List.iter Test_diagnostics.suite ~f:(fun x -> x ())
