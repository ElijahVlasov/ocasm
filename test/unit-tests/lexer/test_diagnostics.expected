----- Incorrect number literal -----

Error: unexpected character 'l' in a number literal.
File "", line = 1, col = 9, line = 1, col = 9.
Error: unexpected character 'z' in a number literal.
File "", line = 2, col = 2, line = 2, col = 2.

--------------

----- Incorrect escaped sequence -----

Error: incorrect escape sequence '\u'.
File "", line = 1, col = 10, line = 1, col = 10.

--------------

